recipes/sentry-native/all/conanfile.py

```diff
@@ -130,6 +130,8 @@ def validate(self):
             raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
 
     def build_requirements(self):
+        if tools.Version(self.version) >= "0.4.0" and self.settings.os == "Windows":
```

Comment:
Currently all version are upper to 0.4.0 so we might remove this:

```suggestion
        if self.settings.os == "Windows":
```